validate/main.go

```diff
@@ -53,13 +53,32 @@ func die(format string, a ...interface{}) {
 	os.Exit(1)
 }
 
-func runIgnValidate(cmd *cobra.Command, args []string) {
+func Usage() {
+	stdout(`
+ignition-validate will validate Ignition configs
+
+Usage:
+  ignition-validate config.ign [flags]
+
+Flags:
```

Comment:
could use https://golang.org/pkg/flag/#PrintDefaults here to avoid needing to duplicate the help text. Note that it prints to stderr by default, so either they should both print to stderr or both to stdout (I don't have a preference which).